### DISCUSSION TITLE
blog/Crafting-your-own-chat-app-using-Hilla-and-Spring-AI

### DISCUSSION THREAD

**Discussion by giscus** [MaheshBabu11/personal-blog#3] [General]
# blog/Crafting-your-own-chat-app-using-Hilla-and-Spring-AI

Creating a chat app using Hilla and Spring AI to interact with an OenAI or Azure OpenAI service. The tutorial covers the use of Hilla components for UI and Spring AI for AI integration.

http://localhost:3000/blog/Crafting-your-own-chat-app-using-Hilla-and-Spring-AI

**Comment by MaheshBabu11** [comment]
this looks cool
